src/Combinatorics.jl

```diff
@@ -1,3 +1,5 @@
+__precompile__()
+
 module Combinatorics
 
 using Compat, Polynomials, Iterators
```

Comment:
was removed at some point from https://github.com/JuliaLang/Combinatorics.jl/blob/master/REQUIRE but not here, which means the package would be broken if there were ever versions of its dependencies that didn't transitively depend on Compat